bootstrap/roles/ocp4-install-signing/tasks/main.yaml

```diff
@@ -0,0 +1,6 @@
+- import_tasks: build-cosign-infra.yaml
```

Comment:
LGTM. Another alternative could be to include in the tekton step to use the cosign step (for example I've used in https://github.com/rcarrata/ocp4-network-security/blob/main/sign-images/manifests/tasks-cosign.yaml#L33)